trestle/tasks/osco_to_oscal.py

```diff
@@ -0,0 +1,184 @@
+# -*- mode:python; coding:utf-8 -*-
+
+# Copyright (c) 2020 IBM Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""OSCAL transformation tasks."""
+
+import configparser
+import logging
+import os
+import json
+import pathlib
+import traceback
+from typing import Optional
+
+import yaml
+
+from trestle.lib import osco
+from trestle.tasks.base_task import TaskBase
+from trestle.tasks.base_task import TaskOutcome
+
+logger = logging.getLogger(__name__)
+
+
+class OscoToOscal(TaskBase):
+    """
+    Task to convert OSCO yaml to OSCAL json.
+
+    Attributes:
+        name: Name of the task.
```

Comment:
So a question: Is OSCO the name for the compliance operator or one that we invented internally?

I think we probably need to spell it out in the comments.